dashboard/app/views/shared/_concept_trophy_block.html.haml

```diff
@@ -1,5 +1,8 @@
 %div{style: 'display: inline-block; width: 290px; position: relative; overflow: hidden;' + added_style}
   %h4 #{t('nav.popup.mastery')}
+  %p
+    %strong Note:
+    These badges will be removed for the 2016-2017 school year.
```

Comment:
These badges are going to be removed, not removed for one school year. Consider "These trophies will be removed August 2016."